packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.template.html

```diff
@@ -0,0 +1,111 @@
+                    data-size="s"
+                    class="ml-2"
+                ></oui-spinner>
+            </oui-datagrid-topbar>
+            <oui-datagrid-column
+                data-property="ip"
+                data-title="'IP'"
+                data-sortable="asc"
+                data-type="string"
+                data-filterable
+            >
+                <span data-ng-if="!$row.isForm" data-ng-bind="$row.ip"></span>
+                <oui-select
+                    data-ng-if="$row.isForm"
+                    data-name="ip"
+                    data-items="$ctrl.authorizedAccesses"
+                    data-match="ip"
+                    data-group-by="'type'"
+                    data-model="$ctrl.model.ip"
+                    data-searchable
+                    data-required
+                >
+                    <span data-ng-bind="$item.ip"></span>
+                </oui-select>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-property="type"
+                data-title=":: ('nasha_dashboard_partition_accesses_list_type' | translate)"
+                data-type="options"
+                data-type-options="$ctrl.datagridTypeOptions"
+                data-filterable
+                data-sortable
+            >
+                <span
+                    data-ng-if="!$row.isForm"
+                    data-translate="{{ 'nasha_dashboard_partition_accesses_list_type_' + $row.type }}"
+                ></span>
+                <oui-select
+                    data-ng-if="$row.isForm"
+                    id="type"
+                    name="type"
+                    data-model="$ctrl.model.type"
+                    data-items="$ctrl.typeOptions"
+                    data-match="label"
+                    data-required
+                >
+                    <span data-ng-bind="$item.label"></span>
+                </oui-select>
+            </oui-datagrid-column>
+            <oui-datagrid-column>
+                <div
+                    data-ng-if="$row.isForm"
+                    class="d-flex justify-content-end align-items-center"
+                >
+                    <oui-spinner
+                        data-ng-if="$ctrl.isCreatingAccess"
+                        data-size="s"
+                        class="mr-2"
+                    ></oui-spinner>
+                    <oui-button
+                        class="mr-2"
+                        data-type="submit"
+                        data-disabled="!$ctrl.canCreateAccess"
+                    >
+                        <span class="oui-icon oui-icon-check"></span>
+                    </oui-button>
+                    <oui-button
+                        data-on-click="$ctrl.hideAccessForm()"
+                        data-disabled="$ctrl.isCreatingAccess || $ctrl.accesses.length === 1"
+                    >
+                        <span class="oui-icon oui-icon-close"></span>
```

Comment:
```suggestion
                        <span class="oui-icon oui-icon-close" aria-hidden="true"></span>
```
also please add a sr-only description if there is no text input

Reply:
:+1: